Service/GithubPubliccodeService.php

```diff
@@ -305,15 +400,297 @@ public function importRepository($repository): ?ObjectEntity
 
         isset($this->io) && $this->io->comment('Checking repository '.$repository['name']);
         $synchronization->setMapping($repositoryMapping);
-        $synchronization = $this->synchronizationService->synchronize($synchronization, $repository);
+        $synchronization = $this->synchronizationService->handleSync($synchronization, $repository);
```

Comment:
Is there a reason to revert to handleSync? The synchronize function should replace handleSync in order to force remapping if the mapping changes.